openedx/features/course_experience/views/course_home.py

```diff
@@ -141,6 +145,11 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         # Get the course tools enabled for this user and course
         course_tools = CourseToolsPluginManager.get_enabled_course_tools(request, course_key)
 
+        # Grab the course home message fragment
+        course_message_fragment = CourseMessageFragmentView().render_to_fragment(
```

Comment:
Nit: maybe the class should be ``CourseHomeMessageFragmentView`` since that's what you wrote in the comment above.